AEPAssurance/Source/AssuranceSessionDetails.swift

```diff
@@ -37,7 +37,7 @@ class AssuranceSessionDetails {
     /// The 4 digit authentication code to connect to a session.
     ///
     /// pinCode is obtained as user input from PinCode screen.
```

Comment:
nit: comment should match variable name :)

Reply:
Good catch. Fixed